hack/ramen-must-gather

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+function usage {
+	echo "ramen-must-gather [appNamespace [hubClusterName [managedClusterNames...]]]"
+	echo "default appNamespace is current namespace"
+	echo "default hubClusterName is 'hub'"
+	echo "default managedClusterNames are all clusters in your kubeconfig"
```

Comment:
This may not work when you have multiple environments (e.g. my-hub, my-dr1, my-dr2, someone-hub, someone-dr1, someone-dr2).